config/sources/families/include/meson64_common.inc

```diff
@@ -226,7 +226,7 @@ family_tweaks_bsp()
 	    EndSection
 	EOF
 
-	elif [[ $BOARD == odroidn2 || $BOARD == odroidc4 || $BOARD == vim2 ]]; then
+	elif [[ $BOARD == odroidn2 || $BOARD == odroidc4 || $BOARD == vim2 || $BOARD == odroidhc4 || $BOARD == khadas-vim3 || $BOARD == khadas-vim3l ]]; then
 
```

Comment:
Long lint in `if`.
I propose to change it to `case` like there https://github.com/armbian/build/blob/fba6379d5262e99b9abd67db94c0772daf1c6f21/config/sources/families/include/rockchip64_common.inc#L16:L23